helm/chaos-mesh/crds/crd.yaml

```diff
@@ -0,0 +1,3507 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We can delete this file. It is duplicate with other files

Reply:
done